components/homme/src/share/prim_driver_base.F90

```diff
@@ -1468,113 +1426,114 @@ subroutine applyCAMforcing_tracers(elem,hvcoord,np1,np1_qdp,dt,adjustment)
                   ! dyn_in%elem(ie)%state%Qdp(i,j,k,ic,tl_fQdp) = &
                   !        dyn_in%elem(ie)%state%Qdp(i,j,k,ic,tl_fQdp) + fq 
                   elem%state%Qdp(i,j,k,ic,np1_qdp) = &
-                  dp(i,j,k)*elem%derived%FQ(i,j,k,ic)
-
+                       dp(i,j,k)*elem%derived%FQ(i,j,k,ic)
+                  
                   if (ic==1) then
```

Comment:
just nitpicky, so feel free to ignore.  But isn't the usage of `ic` effectively the same as `q` in this routine?  Same for `fq` and `v1`?  Could we get away with making these the same to make the loops more readable?

Reply:
agreed.